java/examples/src/main/java/glide/examples/ExamplesApp.java

```diff
@@ -33,6 +35,21 @@ private static void runGlideExamples() {
             System.out.println("SET(apples, oranges): " + client.set("apples", "oranges").get());
             System.out.println("GET(apples): " + client.get("apples").get());
 
+            // When added via GEOADD, the geospatial coordinates are converted into a 52 bit geohash, so
+            // the coordinates
+            // returned might not be exactly the same as the input values
+            client
```

Comment:
It is ok to have it here, but usually we don't update the example.

Reply:
oops, did not mean to include this